include/modem/lte_lc.h

```diff
@@ -455,8 +455,8 @@ enum lte_lc_evt_type {
 	 * @ref lte_lc_rai_cfg in the event.
 	 *
 	 * @note This is only supported by the following modem firmware:
-	 * - mfw_nrf91x1 >= v2.0.2
-	 * - mfw_nrf9151-ntn
+	 *       - mfw_nrf91x1 >= v2.0.2
```

Comment:
```suggestion
	 *       - mfw_nrf91x1 v2.0.2 or later
```
To be consistent with libmodem docs.